library/cohesity_restore_file.py

```diff
@@ -117,6 +122,19 @@
       - protection run timestamp in YYYY-MM-DD:HH:MM format to use as source for the Restore operation. If not specified,
         the most recent timestamp is used
     type: String
+  vm_name:
+    description:
+      - Name of the Vcenter virtual machine, from where the files are located. Required if the environment is VMware.
+    type: String
+  vm_username:
+    description:
+      - Username of the virtual machine, where files will be restored. Required if the environment is VMware.
```

Comment:
Why is password required for VMware VM , this is through vCenter ?

Reply:
Even if we try to do the same through UI we need to provide creds.